Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
Oh, but isn't OpenJDK a bit buggy? I thought it was being "Oracle Java" in terms of testing. I need to have a look at this -> my main concern is where does "java-oracle" comes from? I'm having a look